src/args.rs

```diff
@@ -33,7 +36,11 @@ impl Args {
             } else if let Some(str) = cli.restic_password {
                 Password::Plain(str)
             } else {
-                unreachable!("Error in Config: neither password_command nor password_file found. Please open an issue if you see this.")
+                let pwd = Self::read_password_from_stdin();
```

Comment:
This `else` block started looking a bit too big, but I'm not sure about Rust best practices. Maybe it's overkill to extract a method with four lines (one of which is a comment).